tests/test_build.py

```diff
@@ -0,0 +1,23 @@
+try:
+    from unittest import mock
+except ImportError:
+    # `Python 2` or lower than `Python 3.3` does not
```

Comment:
This reminds me we still need to decide when p4a drops python2 support. Soon would probably be good, it's especially annoying nowadays to do python2 compatibility stuff when it will be necessary for only a short time.

Reply:
Completely agree. Also I can't wait to use f-strings and annotations :smile

Reply:
Yes, I'm also waiting for python 3's `f-strings`, but this will require `python >= 3.6`...**I have no problem with that**

**As a side note:** we could already use python 3 `f-strings` but now it would require the install of some extra python package like [fmt](https://github.com/damnever/fmt) (I've been tempted of doing this in some prs :laughing:)